string[] obj_ids        # in the case where you know the ObjectId from mongodb
time start_time     # start time when upper body was recorded, used when obj_id is empty
time stop_time      # stop time when upper body was recorded, used when obj_id is empty
---
vision_people_logging/LoggingUBD[] logs
